patches/third_party-blink-renderer-core-frame-cached_permission_status.cc.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/third_party/blink/renderer/core/frame/cached_permission_status.cc b/third_party/blink/renderer/core/frame/cached_permission_status.cc
+index e0bf619731dcefa800c1590d334433fcc46f8eaf..c44d6b0754a694c6f9539984f3870e9cba75da7c 100644
+--- a/third_party/blink/renderer/core/frame/cached_permission_status.cc
++++ b/third_party/blink/renderer/core/frame/cached_permission_status.cc
+@@ -15,7 +15,6 @@ using mojom::blink::PermissionDescriptorPtr;
+ using mojom::blink::PermissionName;
+ using mojom::blink::PermissionObserver;
+ using mojom::blink::PermissionService;
+-using mojom::blink::PermissionStatus;
```

Comment:
what is the exact error? The commit message should also link to the previous fix that is referenced

Reply:
Amended the commit here https://github.com/brave/brave-core/pull/26068/commits/335ede28eab543021c73f9d0ad7b7bcf01c95490 to include link to previous fix and error message.

Reply:
I think this can be fixed with a chromium_src override for `ActiveScriptWrappableCreationKey` as mentioned in dm

Reply:
This one is really odd and not sure how to deal with it or why it doesn't show up for upstream.